arm/Microsoft.DocumentDB/databaseAccounts/deploy.bicep

```diff
@@ -50,15 +50,19 @@ param maxIntervalInSeconds int = 300
   '3.2'
   '3.6'
   '4.0'
+  '4.2'
 ])
-param serverVersion string = '4.0'
+param serverVersion string = '4.2'
```

Comment:
General question @itpropro: Do you, by any chance, have a pipeline run/badge available that can show that the changes work? It would, I guess, go along with a new parameter file `gremlin.parameters.json`.

Reply:
I will prepare and test a parameters.json file. I was not sure how the current testing pipeline looks, as there is not really a test framework that can programmatically go though all parameters, right?

Reply:
The pipelines that come with CARML actually test both Static Tests (via Pester) as well as the deployments of all parameter files. So the ask would have been to run the pipeline in your Fork & attach the badge of its run to the PR.

Like for your other PRs I now rebased the code